hw-04-app/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿using SortingAndOutput;
+
+var set = new sbyte[7];
+bool a = true;
+
+ArraysOperations.CreatingArray(set);
+Console.WriteLine("Вывод неотсортированного массива:");
+ArraysOperations.OutputArray(set);
+while (a)
+{
+    ChoosingAction();
+}
+
+void ChoosingAction()
```

Comment:
C# разрешает использовать тут внутреннюю функцию - она имеет доступ к переменной `a` которая не передается через параметр, а находится в тот же области видимости.

иногда это не очень хорошо и может приводить к ошибкам. хотя такие нюансы out of наших лекций

если будет интересно то попробуйте так, что изменится:
```suggestion
static void ChoosingAction()
```

ну и почитать из русскоязычного наверное тут:
https://metanit.com/sharp/tutorial/3.54.php